sdk/containers/azcontainerregistry/authentication_client_test.go

```diff
@@ -8,22 +8,23 @@ package azcontainerregistry
 
 import (
 	"context"
+	"net/http"
```

Comment:
Didn't notice my editor do this, but it might be nice to turn on import sorting!